Submitted by: Jim Gifford (patches at jg555 dot com)
Date: 2006-10-12
Initial Package Version: 5.5
Upstream Status: Sent
Origin: Jim Gifford
Description: Fixes Ncurses to work with Bash 3.2

diff -Naur ncurses-5.5.orig/include/MKhashsize.sh ncurses-5.5/include/MKhashsize.sh
--- ncurses-5.5.orig/include/MKhashsize.sh	1996-05-11 06:17:23.000000000 -0700
+++ ncurses-5.5/include/MKhashsize.sh	2006-10-12 07:20:38.389473829 -0700
@@ -7,7 +7,7 @@
 echo " */"
 
 CAPS="${1-Caps}"
-TABSIZE=`grep -v '^[ #]' $CAPS | grep -v "^$" | grep -v "^capalias"| grep -v "^infoalias" | wc -l`
+TABSIZE="`grep -v '^[ #]' $CAPS | grep -v '^$' | grep -v '^capalias' | grep -v '^infoalias' | wc -l`"
 
 echo ""
 echo "#define CAPTABSIZE	${TABSIZE}"
